debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -763,6 +781,11 @@ public OffsetStorageReader offsetStorageReader() {
                             try {
                                 if (changeRecords != null && !changeRecords.isEmpty()) {
                                     logger.debug("Received {} records from the task", changeRecords.size());
+                                    changeRecords = changeRecords.stream().map(transformations::transform).filter(x -> x != null).collect(Collectors.toList());
```

Comment:
Could you please move the stream operators (filter, map, collect) to a new line each? Otherwise it's a nightmare to debug.

Reply:
Done